vllm/engine/async_llm_engine.py

```diff
@@ -1,7 +1,7 @@
 import asyncio
 import time
 from functools import partial
-from typing import Any, Dict, Iterable, List, Optional, Set, Tuple, Type, Union
+from typing import Any, Callable, Coroutine, Dict, Iterable, List, Optional, Set, Tuple, Type, Union
```

Comment:
Nit: add parenthesis around this long import and then run the formatter.
```suggestion
from typing import (Any, Callable, Coroutine, Dict, Iterable, List, Optional, Set, Tuple, Type, Union)
```